src/pages/home/sidebar/ChatSwitcherView.js

```diff
@@ -160,6 +160,15 @@ class ChatSwitcherView extends React.Component {
             .map((report) => {
                 const participants = lodashGet(report, 'participants', []);
                 const isSingleUserDM = participants.length === 1;
+                const personalDetails = this.props.personalDetails
+                    ? this.props.personalDetails
+                    : {};
+                const participant1Avatar = personalDetails[participants[0]]
```

Comment:
We don't seem to be using this style for this type of conditions. I would just use something like:
``` javascript
const participant1Avatar = personalDetails[participants[0]]
    ? personalDetails[participants[0]].avatarURL : '';
```
For some reason the style you used feels awkward to read, probably because I'm not use to it. And the one I suggest feels more common on our code.

Reply:
Okay. Fixed it.